snippets/cs.snippets

```diff
@@ -103,15 +103,15 @@ snippet for
 	}
 # foreach
 snippet fore
-	foreach (var ${1:entry} in ${2}) {
+	foreach (${1:var} ${2:entry} in ${3}) {
```

Comment:
no, it is becouse you can use var or specify the variable type, it does not work without it.
like I wrote it it works the same as it does in visual studio.